etc/profile-m-z/nano.profile

```diff
@@ -47,7 +47,7 @@ x11 none
 private-bin nano,rnano
 private-cache
 private-dev
-# Comment the next line if you want to edit files in /etc directly
+# Add 'ignore private-etc' to your nano.local if you want to edit files in /etc directly.
 private-etc alternatives,nanorc
```

Comment:
OT: Does this work w/o `writeable-etc`?

Reply:
Nice catch, forgot I have that in my nano.local. For good measure I added a similar comment for editing files in /var too. What do you think?

Reply:
Moved the writable-var comment cfr. profile.template. Thanks for your thorough review!